README.md

```diff
@@ -28,15 +28,18 @@ All new backend code and React components should be well tested.
 
 ## Supported operating systems
 
-* Ubuntu 18.04, 20.04, 22.04 or equivalent
+* Ubuntu 18.04, 20.04 or equivalent
+* [Manjaro](manjaro.md)
```

Comment:
I think we should move this somewhere else. This sort of implies that there's a reasonable assumption that Manjaro will continue working over time. I'd love if it does but we don't have the resources to make that "guarantee" to people. 

Perhaps we add this below the Ubuntu line:



```markdown

The following operating systems are unofficially supported but aren't regularly reviewed

* [Manjaro](manjaro.md)
```